src/toil_lib/tools/quantifiers.py

```diff
@@ -82,46 +82,36 @@ def run_rsem(job, bam_id, rsem_ref_url, paired=True):
+def run_rsem_postprocess(job, rsem_gene_id, rsem_isoform_id):
     """
     Parses RSEMs output to produce the separate .tab files (TPM, FPKM, counts) for both gene and isoform.
     These are two-column files: Genes and Quantifications.
     HUGO files are also provided that have been mapped from Gencode/ENSEMBLE names.
 
     :param JobFunctionWrappingJob job: passed automatically by Toil
-    :param str uuid: UUID to mark the samples with
     :param str rsem_gene_id: FileStoreID of rsem_gene_ids
     :param str rsem_isoform_id: FileStoreID of rsem_isoform_ids
     :return: FileStoreID from RSEM post process tarball
     :rytpe: str
     """
     work_dir = job.fileStore.getLocalTempDir()
     # I/O
-    job.fileStore.readGlobalFile(rsem_gene_id, os.path.join(work_dir, 'rsem_gene.tab'), mutable=True)
-    job.fileStore.readGlobalFile(rsem_isoform_id, os.path.join(work_dir, 'rsem_isoform.tab'), mutable=True)
-    # Convert RSEM files into individual .tab files.
-    docker_call(job=job, tool='jvivian/rsem_postprocess', parameters=[uuid], work_dir=work_dir)
-    os.rename(os.path.join(work_dir, 'rsem_gene.tab'), os.path.join(work_dir, 'rsem_genes.results'))
-    os.rename(os.path.join(work_dir, 'rsem_isoform.tab'), os.path.join(work_dir, 'rsem_isoforms.results'))
-    output_files = ['rsem.genes.norm_counts.tab', 'rsem.genes.raw_counts.tab', 'rsem.isoform.norm_counts.tab',
-                    'rsem.isoform.raw_counts.tab', 'rsem_genes.results', 'rsem_isoforms.results']
+    genes = job.fileStore.readGlobalFile(rsem_gene_id, os.path.join(work_dir, 'rsem_genes.results'), mutable=True)
+    iso = job.fileStore.readGlobalFile(rsem_isoform_id, os.path.join(work_dir, 'rsem_isoforms.results'), mutable=True)
     # Perform HUGO gene / isoform name mapping
-    genes = [x for x in output_files if 'rsem.genes' in x]
-    isoforms = [x for x in output_files if 'rsem.isoform' in x]
-    command = ['-g'] + genes + ['-i'] + isoforms
-    docker_call(job=job, tool='jvivian/gencode_hugo_mapping', parameters=command, work_dir=work_dir)
-    hugo_files = [os.path.splitext(x)[0] + '.hugo' + os.path.splitext(x)[1] for x in genes + isoforms]
+    command = ['-g', genes, '-i', iso]
+    docker_call(job=job, tool='quay.io/ucsc_cgl/gencode_hugo_mapping:1.0--cb4865d02f9199462e66410f515c4dabbd061e4d',
+                parameters=command, work_dir=work_dir)
+    hugo_files = [os.path.join(work_dir, x) for x in ['rsem_genes.hugo.results', 'rsem_isoforms.hugo.results']]
     # Create tarballs for outputs
-    tarball_files('rsem.tar.gz', file_paths=[os.path.join(work_dir, x) for x in output_files], output_dir=work_dir)
-    tarball_files('rsem_hugo.tar.gz', [os.path.join(work_dir, x) for x in hugo_files], output_dir=work_dir)
+    tarball_files('rsem.tar.gz', file_paths=[os.path.join(work_dir, x) for x in [genes, iso]], output_dir=work_dir)
+    tarball_files('rsem_hugo.tar.gz', file_paths=[os.path.join(work_dir, x) for x in hugo_files], output_dir=work_dir)
```

Comment:
Where do these hugo files come from? The hugo_files list looks like it's paths that are assumed to exist. Does tarball_files fail clearly if those files aren't there?